docs/api/settings.rst

```diff
@@ -86,3 +87,17 @@ The application name.
 
 This should be unique to all the services running in the application ecosystem. It is used by
 the LoggingMiddleware and Prometheus integration.
+
+
+``ENCODER``
+------------------
+
+**Optional**
+
+Default: `rest_framework.utils.encoders.JSONEncoder <https://github.com/encode/django-rest-framework/blob/master/rest_framework/utils/encoders.py#L17>`_
+
+`Encoder class <https://docs.python.org/3/library/json.html#json.JSONEncoder>`_ to use for
+serializing your Python data structure to a json object when publishing.
+
+NOTE: The default encoder class is subject to change in an upcoming release. It is
+strongly advised that you use this setting explicitly.
```

Comment:
We do not really need to set it explicitly *unless* you depend on a superset of the standard JSON encoder.

Reply:
Right. In most cases we will fall back to the optional default. But, as of now we are guaranteeing that `serializer.data` can be serialized via a superset of JSONEncoder. And as far as I know, we won't in the future. If set, it would allow breaking changes, to be less severe.  But if you think its too much overhead, then Ill change it.

Reply:
Unless of course, we want to set it by default to `json.encoder.JSONEncoder`. It would be the eventual step that needs to be taken. I guess do we do it now, or later?